src/java_bytecode/java_class_loader.cpp

```diff
@@ -91,11 +91,14 @@ optionalt<java_bytecode_parse_treet> java_class_loadert::get_class_from_jar(
   debug()
     << "Getting class `" << class_name << "' from JAR " << jar_file << eom;
 
-  std::string data =
+  auto data =
     jar_pool(class_loader_limit, jar_file).get_entry(jar_index_it->second);
 
+  if(!data.has_value())
+    return {};
```

Comment:
Previous behaviour here was to terminate the program and now returns an empty parse tree. This seems reasonable but I wonder if this now hides typos?

Reply:
This isn't an empty parse tree, but a nullopt one -- very different from an empty string!

Note that the sole user checks existence of the file before calling, i.e., this should be unreachable.